lisa/base_tools/wget.py

```diff
@@ -84,8 +84,7 @@ def get(
         actual_file_path = self.node.execute(
             f"ls {download_file_path}", shell=True, sudo=sudo
         )
-        if actual_file_path.exit_code != 0:
-            raise LisaException(f"File {actual_file_path} doesn't exist.")
+        actual_file_path.assert_exit_code(0, f"File {actual_file_path} doesn't exist.")
```

Comment:
Thanks for helping with this. This needs to be moved into `node.execute`. Please look at the following arguments in the `node.execute` function: `expected_exit_code`, `expected_exit_code_failure_message`. Also, please look at my comment in #1804. Similar comments apply to other changes in this PR.

Reply:
For example, this is what should go here:
```
actual_file_path = self.node.execute(
             f"ls {download_file_path}", shell=True, sudo=sudo,
             expected_exit_code=0,
             expected_exit_code_failure_message=f"File path doesn't exist"
         )
```

Reply:
Wouldn't it be more difficult to debug without the 'actual_file_path' displayed in the log?

Should the same pattern be followed for the messages that includes result.stderr and result.stdout? ([ex](https://github.com/microsoft/lisa/pull/2196/files#diff-deecb50c83f417b24953881efd72fb99a30de400c8caa03c3750b9950fb14063R807))

Reply:
Maybe you can add `download_file_path` to the logs. That should be enough. We also have some logs thrown from `node.execute` which should be enough to debug this issue.

Reply:
As discussed, please follow @squirrelsc suggestion related to stderr/stdout logging in PR #2083